### PR TITLE
Add option for 'lite' simulations

### LINKED ISSUE #2027: Add option for 'lite' simulations
By default, we could run simulations on a smaller sample of each dataset.

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Fixes #2027

Open to feedback or rejection.

https://github.com/user-attachments/assets/b802286c-2a1e-41ca-b474-e25e6d907c52

### CODE REVIEW

**Comment by anth-volk** [comment]
Would you mind re-uploading the video? Doesn't display for me.

**Comment by anth-volk** [comment]
Also, there's a bug whereby you used loose equality (`==`) instead of strict (`===`)

File: src/pages/policy/output/FetchAndDisplayImpact.jsx [hunk 1]
```diff
@@ -36,6 +36,7 @@ export function FetchAndDisplayImpact(props) {
   const timePeriod = searchParams.get("timePeriod");
   const reformPolicyId = searchParams.get("reform");
   const baselinePolicyId = searchParams.get("baseline");
+  const maxHouseholds = searchParams.get("mode") == "full" ? null : 10_000;
```
Comment (anth-volk) [inline src/pages/policy/output/FetchAndDisplayImpact.jsx:39]:
```suggestion
  const maxHouseholds = searchParams.get("mode") === "full" ? null : 10_000;
```

**Comment by anth-volk** [comment]
I think the "lite" simulations option looks great, but is the timing component working correctly? It doesn't appear in the video, but perhaps it was added after that was taken.

**Comment by anth-volk** [comment]
Curious to hear Max's thoughts

**Comment by MaxGhenis** [comment]
Can we start by making lite the non-default option? Including rephrasing the toggle as such. In general looking to avoid changing default behaviors unless necessary--we should give users some sort of notice analogous to deprecation warnings.

**Comment by anth-volk** [comment]
@MaxGhenis Correct me if I'm wrong, Nikhil, but my understanding on the states is that their datasets are too small to be affected by the toggle.

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**